### ISSUE TITLE
How to do Batch Normalization in Flax?

### ISSUE THREAD

**Issue by marcvanzee** [google/flax#855]

**Comment by marcvanzee** [comment]
Closing this since I expect users will be able to search our examples for BatchNorm, and if not we can point them to it. This doesn't seem to require a HOWTO.
